pkg/fetcher/fetcher.go

```diff
@@ -100,6 +100,21 @@ type URLFetcher struct {
 	options Options
 }
 
+// ErrorResponseBody struct
+type ErrorResponseBody struct {
+	Errors []struct {
+		Code string
+		Message string
+
+		Detail []struct {
```

Comment:
My bad - rechecked the documentation, `Detail` is actually unstructured. I could drop the `Detail` field for now (since only `Message` field is used, and pattern of `Detail` is undetermined)